components/atoms/Accordion/Accordion.js

```diff
@@ -0,0 +1,55 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion({ title, summary, description }) {
```

Comment:
We should pass in the description as `children` because we are going to end putting pretty varied content in here.